builtin/logical/database/path_roles_test.go

```diff
@@ -154,7 +337,7 @@ func TestBackend_StaticRole_Config(t *testing.T) {
 
 			expected := tc.expected
 			actual := make(map[string]interface{})
-			for _, key := range dataKeys {
+			for _, key := range []string{"username", "password", "last_vault_rotation", "rotation_period"} {
```

Comment:
Nit: Should we keep `dataKeys := []string{...}` but have it be local to `TestBackend_StaticRole_Config`?

Reply:
Will do 👍